scripts/create_newcase

```diff
@@ -52,8 +52,9 @@ def parse_command_line(args, cimeroot):
     parser.add_argument("--project", "-project",
                         help="Specify a project id")
 
-    parser.add_argument("--pecount", "-pecount", choices=('S','M','L','X1','X2'), default="M",
-                        help="Specify a target size description for the number of cores")
+    parser.add_argument("--pecount", "-pecount",  default="M",
```

Comment:
Needed to add support for the TASKSxCOUNT option here.  It was previously handled in create_test after create_newcase was completed, but I ran into a problem where the default pelayout was too big for the test system which caused create_newcase to throw an error before the pelayout adjustment specified in the test could be applied.    The choices option to add_argument was a nice feature, but I couldn't figure out how to use it with these additional possibilities.